src/Shared/Microsoft.DotNet.GitHub.Authentication/GitHubAppTokenProvider.cs

```diff
@@ -3,12 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using GitHubJwt;
+using Microsoft.DotNet.GitHub.Authentication;
```

Comment:
Is this using necessary?

Reply:
Was it merged with the using statement? this is something that should probably be cleaned up as it seems weird to add a using for the same namespace that the class is in.

Reply:
Sorry my email on the comment came later. :( I will clean it up.

Reply:
Actually those are 2 different ones 

1) Microsoft.DotNet.GitHub.Authentication;
2) Microsoft.Dotnet.GitHub.Authentication

Reply:
There is this this weird naming we have all over.
Microsoft.DotNet.GitHub.Authentication has all our classes .  Microsoft.Dotnet.GitHub.Authentication is Github lib namespace. So it looks like they are the same but they actually are not.  :)

Reply:
Is that just a typo in this file?

Reply:
Because this class is ours and it's in the Dotnet namespace

Reply:
Ooh, maybe the interface is in the wrong namespace then? and it should be using the lowercase one?

Reply:
I see some files have DotNet other have Dotnet. Its all over in that folder. Need to fix that. I can create another PR to fix this.